data-collection/deploy/module-trusted-advisor.yaml

```diff
@@ -202,6 +203,8 @@ Resources:
                           resource["Status"] = resource.pop("status") if "status" in resource else '-'
                           output.update({"AccountId":account_id, "AccountName":account_name, "Category": check["category"], 'DateTime': dt, 'Timestamp': ts, "CheckName": check["name"], "CheckId": check["id"]})
                           output.update(resource)
+                          translator = str.maketrans(' ', '_', string.punctuation)
```

Comment:
I don't think spaces is a problem. I suggest to exclude this line as it will also break TAO where we have spaces in schema - https://github.com/aws-samples/aws-cudos-framework-deployment/blob/main/cid/builtin/core/data/queries/tao/glue_table.json. The issue with duplicating columns were caused by difference in case

Reply:
Done, keeping only lowercase and I reviewed these and I do not see obvious conflicts with https://github.com/aws-samples/aws-cudos-framework-deployment/blob/main/cid/builtin/core/data/queries/tao/glue_table.json.